src/odb/include/odb/db.h

```diff
@@ -3190,24 +3201,30 @@ class dbInst : public dbObject
   /// If false, it will be added to the top module.
   /// Returns nullptr if an instance with this name already exists.
   /// Returns nullptr if the master is not FROZEN.
-  ///
+  /// If dbmodule is non null the dbInst is added to that module.
```

Comment:
If null it should be added to the top module.  There should be no instances not in some module.